### PR TITLE
Make the Free plan tagline on My Plan page dependant on Backup product

### PR DESCRIPTION
**Pull request by delawski**
The Free plan tagline on My Plan page is always the same - no matter if the user already purchased one of the Jetpack Backup options. This can be confusing since the default tagline prompts the user to purchase a paid plan in order to get backups while the user might already have a separate Jetpack Backup product.

This PR resolves this issue. The Free plan tagline is now different depending if the user has one of the Jetpack Backup options or no additional product.

**Before**
Screenshot 2019-11-26 at 16 48 12

**After - without Jetpack Backup**
Screenshot 2019-11-26 at 16 30 50

**After - with Jetpack Backup Daily**
Screenshot 2019-11-26 at 16 17 14
 
**After - with Jetpack Backup Real-Time**
Screenshot 2019-11-26 at 16 30 06

#### Changes proposed in this Pull Request

* Make the Free plan tagline on My Plan page dependant on Backup product

#### Testing instructions

* Go to http://calypso.localhost:3000/plans/my-plan and select a Jetpack site on the Free plan
* Confirm that the tagline is correct depending if you have no Jetpack Backup, Backup Daily or Backup Real-Time.

Fixes n/a

### CODE REVIEW

File: client/lib/plans/plans-list.js [hunk 1]
```diff
@@ -641,10 +641,24 @@ export const PLANS_LIST = {
...
-		getTagline: () =>
-			i18n.translate(
-				'Upgrade your site to access additional features, including spam protection, backups, and priority support.'
-			),
+		getTagline: feature => {
+			switch ( feature ) {
+				case constants.FEATURE_JETPACK_BACKUP_DAILY:
+				case constants.FEATURE_JETPACK_BACKUP_DAILY_MONTHLY:
+					return i18n.translate(
+						'Upgrade your site to access additional features, including spam protection, real-time backups, and priority support.'
```
Comment (a8ci18n) [inline client/lib/plans/plans-list.js]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Upgrade your site to access additional features, including spam protection, backups, and priority support.' )` *ES Score: 8*

File: client/lib/plans/plans-list.js [hunk 2]
```diff
@@ -641,10 +641,24 @@ export const PLANS_LIST = {
...
+					);
+				case constants.FEATURE_JETPACK_BACKUP_REALTIME:
+				case constants.FEATURE_JETPACK_BACKUP_REALTIME_MONTHLY:
+					return i18n.translate(
+						'Upgrade your site to access additional features, including spam protection, and priority support.'
```
Comment (a8ci18n) [inline client/lib/plans/plans-list.js]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Upgrade your site to access additional features, including spam protection, backups, and priority support.' )` *ES Score: 9*

File: client/my-sites/plans/current-plan/purchases-listing.jsx [hunk 3]
```diff
@@ -80,6 +80,13 @@ class PurchasesListing extends Component {
 		return product.expiryMoment < moment().add( 30, 'days' );
 	}
 
+	getJetpackBackup() {
```
Comment (tyxla) [inline client/my-sites/plans/current-plan/purchases-listing.jsx]: I wonder if the method name could be a bit more descriptive? Is it getting the product object, or the raw product, or the purchase? Maybe `getJetpackBackupPurchase` would make more sense?

File: client/my-sites/plans/current-plan/purchases-listing.jsx [hunk 4]
```diff
@@ -102,10 +109,12 @@ class PurchasesListing extends Component {
 			return getJetpackProductTagline( purchase );
 		}
 
+		const jetpackBackup = this.getJetpackBackup();
```
Comment (tyxla) [inline client/my-sites/plans/current-plan/purchases-listing.jsx]: Same comment regarding naming applies here - should `jetpackBackup` be `jetpackBackupPurchase`?

**Comment by delawski** [comment]
@tyxla Thank you for having a look. I've addressed your feedback. Do you mind having another look before merging?

**Pull request merged**